cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -43,6 +44,20 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
       query("SHOW DEFAULT DATABASE", assertDatabaseShown("neo4j")) {
         resultTable()
       }
+      p("It is also possible to filter and sort the results by using `YIELD`, `ORDER BY` and `WHERE`")
+      query("SHOW DATABASES YIELD name, currentStatus, requestedStatus ORDER BY name WHERE name = 'neo4j'", assertDatabaseShown("neo4j")) {
```

Comment:
Ordering by the same column we have an equality predicate on seems like an odd example. Perhaps use a different predicate? What about `currentStatus <> requestedStatus`, or `name CONTAINS 'e'`?

Reply:
I used both, but went with `name CONTAINS 'j'` - otherwise it won't do any filtering as both 'system' and 'neo4j' contain 'e'

Reply:
Or we add a few more databases so the test is a bit more useful?

Reply:
I've added  a couple more databases and a section for SHOW / WHERE. Should be okay now.